SofaKernel/modules/SofaBaseTopology/TopologyDataHandler.inl

```diff
@@ -60,11 +60,11 @@ void TopologyDataHandler <TopologyElementType, VecT>::add(const sofa::helper::ve
     unsigned int i0 = (unsigned)data.size();
     if (i0 != index[0])
     {
-        this->m_topologyData->getOwner()->serr << "TopologyDataHandler SIZE MISMATCH in Data "
+        msg_error(this->m_topologyData->getOwner()) << "TopologyDataHandler SIZE MISMATCH in Data "
             << this->m_topologyData->getName() << ": " << nbElements << " "
             << core::topology::TopologyElementInfo<TopologyElementType>::name()
             << " ADDED starting from index " << index[0]
-            << " while vector size is " << i0 << this->m_topologyData->getOwner()->sendl;
+            << " while vector size is " << i0;
```

Comment:
This is now triggered as an error by AddingTetra2TriangleProcess.scn
See https://ci.inria.fr/sofa-ci-dev/job/sofa-framework/job/PR-1237/CI_CONFIG=ubuntu_gcc-5.4,CI_PLUGINS=options,CI_TYPE=release/4/warnings3Result/